cmake/FileUtil.cmake

```
@@ -26,8 +26,16 @@ function(ly_include_cmake_file_list file)
     include(${file})
     get_filename_component(file_path "${file}" PATH)
     if(file_path)
-        list(TRANSFORM FILES PREPEND ${file_path}/)
+        foreach(f ${FILES})
+            cmake_path(IS_RELATIVE f is_relative)
+            if(is_relative)
+                string(PREPEND f ${file_path}/)
+            endif()
```

CMake [cmake_path(APPEND)](https://cmake.org/cmake/help/latest/command/cmake_path.html#append) function works just like Python pathlib `operator/` so it should be used instead.

If both paths are absolute, the second path replaces the first.
```suggestion
            cmake_path(APPEND file_path ${f} OUTPUT_VARIABLE f)
```